docs/analytics/apis/lookup-tables-api.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Lookup Table API
+description: The Lookup Table API lets you create, get, update, and delete lookup tables to augment your data.
+---
+
+!!!beta "This feature is in beta"
+    This requires that you have the `lookup_table` feature enabled in Amplitude. Ask your rep to help you with this.
+
+### Introduction
+
+Lookup Tables is a way to augment properties. Instead of using formulas, you can upload a CSV file that contains property mappings to derive new properties.
+
+To create a Lookup property, you need to create a Lookup Table to reference. You can retrieve and update each of the tables using the API. Lookup Tables are identified by the name and are scoped per project.
+
+### Create a Lookup Table
+
+To create a Lookup Table, you create a Lookup Table object by uploading a CSV mapping from a property to the new properties to create. You'll need to send a request of type multipart/form-data.
```

Comment:
@dnchn I need clarity here: to create a lookup table, the need to send a CSV via request of the multipart/form-data type that maps old properties to the new properties that should be created? 
If so, maybe we can roll with something like this: 

```suggestion
Create a Lookup Table object by uploading a CSV that maps existing properties to the new properties to create. Send the request with the type multipart/form-data type.
```

Reply:
@dnchn this is the only outstanding thing to get this merged in.

Reply:
Ah, sorry, I missed this. The suggestion is close, but it can only map a single existing property to the new properties.